_sass/epic.scss

```diff
@@ -32,6 +32,12 @@ body > div > div > section > header#page {
     font-size: 22px;
     font-weight: 400;
   }
+  time {
+    font-size: 22px;
+    font-weight: 400;
+    padding-top: 5px;
+    color: #999;
```

Comment:
@eessex I also waffled on this colour, I think you're right though: 

<img width="982" alt="Screen Shot 2019-05-15 at 4 47 18 PM" src="https://user-images.githubusercontent.com/498212/57808356-2c30a880-7731-11e9-8305-dd26135b1f72.png">

```suggestion
```